Demo/app/src/main/java/com/krunal/demo/kotlinPractice/oop/classAndObj/Person2.kt

```diff
@@ -0,0 +1,22 @@
+package com.krunal.demo.kotlinPractice.oop.classAndObj
+
+class Person2(val name: Name) {
+
+    var age: Int = 20
+
+    class Name(
+        var fname: String, var mname: String?, var lname: String
+    ) {
+
+        init {
+//            println("age: $age")
```

Comment:
remove

Reply:
this code is for reference that properties of outer class is not accessible in sub-class without `inner`.